arm-redis/2016-04-01/examples/RedisCacheFirewallRuleCreate.json

```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "ruleName": "rule1",
+    "cacheName": "cache1",
+    "resourceGroupName": "rg1",
+    "api-version": "2016-04-01",
+    "subscriptionId": "subid",
+    "parameters": {
+      "properties": {
+        "startIP": "192.168.1.1",
```

Comment:
Property names should follow camel case style: "startIp" , similarly for "endIp"